html-css/README.md

```diff
@@ -0,0 +1,97 @@
+# HTML & CSS3 Course
```

Comment:
Great Readme file. It would really be helpful for reviewer if the readme was about the website.